resources/lang/en.json

```
@@ -123,6 +123,7 @@
     "instant_build": "Instant build",
     "infinite_gold": "Infinite gold",
     "infinite_troops": "Infinite troops",
+    "max_timer": "Max timer (minutes)",
```

I'd prefer calling this "Game length limit (minutes)" which communicates more clearly what it does. Very subjective tho

---

```suggestion
    "max_timer": "Game length (minutes)",
```
I don't think "limit" is a necessary word, but I agree that game length makes more sense than max timer.